packages/calypso-e2e/src/lib/pages/domains-page.ts

```diff
@@ -0,0 +1,92 @@
+import { Page } from 'playwright';
+
+const selectors = {
+	// Cart popover
+	cartButton: '.popover-cart button',
+	popOver: '.popover',
```

Comment:
nit: I believe it's a single word ([popover](https://en.wikipedia.org/wiki/Popover_(GUI)))

```suggestion
	popover: '.popover',
```

Reply:
Got it!